layouts/partials/footer.html

```diff
@@ -42,7 +43,9 @@
             <li class="sitemap-head" style="padding-top: 30px;">Other Projects</li>
             <li><a href="https://libp2p.io/">libp2p</a></li>
             <li><a href="https://ipld.io/">IPLD</a></li>
+            <li><a href="https://drand.love/">Drand</a></li>
             <li><a href="https://multiformats.io/">Multiformats</a></li>
+            <li><a href="https://testground.ai/">Testground</a></li>
```

Comment:
```suggestion
            <li><a href="http://testground.ai/">Testground</a></li>
```
Turns out we don't have https set up to redirect to anything. I'll ping infra about that but in the meantime ...